ui/docs/reference/workers/docker-worker/capabilities.mdx

```diff
@@ -104,4 +104,4 @@ payload:
 If specified, this configuration will take priority over the `containerEngine` worker config option within Generic Worker.
 Generic Worker defaults to `docker` and this payload option does not have a default.
 
-Note: D2G needs to be enabled with the config option `enableD2G` in Generic Worker for this to work.
+Note: D2G needs to be enabled with the config option `d2GConfig.enableD2G` in Generic Worker for this to work.
```

Comment:
```suggestion
Note: D2G needs to be enabled with the config option `d2gConfig.enableD2G` in Generic Worker for this to work.
```

is it `d2GConfig` or `d2gConfig`? I see second one in the Readme below

Reply:
Great 🦅 👁️  thank you!